portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,76 @@
+body {
```

Comment:
Never use a tag or an Id for CSS selection. Always use CSS classes. In this case, add a class to the body tag and use the class here. Also note that the class name should be in title case. Example `MyBody`.